docs/docs/workflows/create.md

```diff
@@ -29,6 +32,51 @@ For details on the flags, consult the command help via `constellation create -h`
 
 *create* stores your cluster's state into a [`terraform.tfstate`](../architecture/orchestration.md#cluster-creation-process) file in your workspace.
 
+</tabItem>
+<tabItem value="terraform" label="Terraform">
+
+:::caution
+
+There is no extensive input validation with Terraform. If unsure, please use the Constellation CLI instead.
+
+:::
+
+Constellation also supports managing the infrastructure via Terraform. This allows for an easier GitOps integration as well as meeting regulatory requirements.
+Since the Constellation CLI also uses Terraform under the hood, the same Terraform files can be reused.
+For now, please try to refrain from changing the Terraform resource definitions, as Constellation depends on certain components.
+
+Download the Terraform files for the selected CSP from our [GitHub repository](https://github.com/edgelesssys/constellation/tree/main/cli/internal/terraform/terraform).
```

Comment:
Do I have to download all files including subdirectories? This is tedious on the GitHub UI. Any idea for a better UX here? (I once used [curl for this](https://github.com/edgelesssys/constellation/tree/main/cli/internal/terraform/terraform/iam/aws), but with subdirectories it's more complicated.)

Reply:
Yes, basically every time I had the problem I resorted to cloning the entire repository instead. 

We could show those steps. From experience: In the one case we had, the user was able to do this step on their own.

Reply:
Ok, probably not worth to extend this then.